Multiple errors on the code generated by generate screen command

---
What's going on?
Multiple errors on the code generated by the CLI. 

> ignite generate screen login

Executing above command generate following,
ignite-project > src  > views > account > login 
-----------------------------------------------├── index.ts
-----------------------------------------------├── login-screen.tsx

### Generated Files
#### index.ts
`
export * from "./login"
`
> Error: Cannot find module './login'

Fix: Correct path

`
export * from "./login-screen"
` 

#### login-screen.tsx

```
import * as React from "react"
import { Text } from "../../shared/text"
import { NavigationScreenProps } from "react-navigation"
import { Screen } from "../../shared/screen"

export interface Login extends NavigationScreenProps<{}> {
}

// @inject("mobxstuff")
@observer
export class Login extends React.Component<RegisterScreenProps, {}> {
  render () {
    return (
      <Screen preset="fixedCenter">
        <Text preset="header" tx="Login.header" />
      </Screen>
    )
  }
}
```
`
Errors: 
> 1. Cannot find name 'observer'

Fix: include missed import

`import { observer } from "mobx-react";`
> 2. Cannot find name 'RegisterScreenProps'

Fix: Rename interface Login to LoginScreenProps and replace 'RegisterScreenProps' with the same.

### Modified files
#### root-navigator.ts
```
import { StackNavigator } from "react-navigation"
import Login from "../views/account/login/login-screen"
import { ExampleNavigator } from "./example-navigator"

export const RootNavigator = StackNavigator(
  {
    LoginNavigator: { screen: Login },
    exampleStack: { screen: ExampleNavigator },
  },
  {
    headerMode: "none",
    navigationOptions: { gesturesEnabled: false },
  },
)

```
Error:
> Module 'path\to\login-screen' has no default export.

Fix: replace

`import Login from "../views/account/login/login-screen"`

with

`import { Login}  from "../views/account/login/login-screen"`

---
Steps to reproduce
```
$ ignite new PizzaApp
  ( Choose `Bowser (alpha)` when prompted )
$ cd PizzaApp
(create a folder named account under views)
$ ignite generate screen login
( Choose `account` domain when prompted. )
```
---
`ignite doctor` results:
System
  platform           win32

  arch               x64

  cpu                8 cores
          Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  directory          C:\Abhith\Workspaces\Piggy\React Native\Code\v6\piggy-react-native

JavaScript
  node               9.4.0        C:\Program Files\nodejs\node.EXE
  npm                5.6.0        C:\Program Files\nodejs\npm.CMD
  yarn               1.5.1        C:\Users\abhit\AppData\Roaming\npm\yarn.CMD

React Native
  react-native-cli   2.0.1
  app rn version     0.55.1

Ignite
  ignite             2.1.0        C:\Users\abhit\AppData\Roaming\npm\ignite.CMD


Android
  java               -            C:\Program Files (x86)\Common Files\Oracle\Java\javapath\java.EXE
  android home       -            undefined

--- a/commands/screen.js
+++ b/commands/screen.js
@@ -31,7 +31,7 @@ module.exports = async function (context) {
 
   const screenName = name.endsWith('-screen') ? name : `${name}-screen`
   const pascalName = pascalCase(name)
-  const props = { name, pascalName }
+  const props = { name: screenName, pascalName }
 
   const jobs = [
     {
@@ -51,7 +51,7 @@ module.exports = async function (context) {
   if (config.navigation === 'react-navigation') {
     const screenNavigator = pascalName + 'Navigator'
     const appNavFilePath = `${process.cwd()}/src/navigation/root-navigator.ts`
-    const importToAdd = `import ${pascalName} from "../views/${domainPath}${name}/${screenName}"`
+    const importToAdd = `import {${pascalName}} from "../views/${domainPath}${name}/${screenName}"`
     const routeToAdd = `    ${screenNavigator}: { screen: ${pascalName} },`
 
     if (!filesystem.exists(appNavFilePath)) {

--- a/templates/screen.ejs
+++ b/templates/screen.ejs
@@ -1,14 +1,15 @@
 import * as React from "react"
+import { observer } from "mobx-react"
 import { Text } from "../../shared/text"
 import { NavigationScreenProps } from "react-navigation"
 import { Screen } from "../../shared/screen"
 
-export interface <%= props.pascalName %> extends NavigationScreenProps<{}> {
+export interface <%= props.pascalName %>ScreenProps extends NavigationScreenProps<{}> {
 }
 
 // @inject("mobxstuff")
 @observer
-export class <%= props.pascalName %> extends React.Component<RegisterScreenProps, {}> {
+export class <%= props.pascalName %> extends React.Component<<%= props.pascalName %>ScreenProps, {}> {
   render () {
     return (
       <Screen preset="fixedCenter">

